MediaGalleryUi/Model/Config.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryUi\Model;
+
+use Magento\Framework\App\Config\ScopeConfigInterface;
+
+/**
+ * Class Config
+ */
+class Config
+{
+    /**
+     * Path to enable/disable media gallery in the system settings.
+     */
+    private const XML_PATH_ENABLED = 'media/gallery/enabled';
```

Comment:
Please update the path to (also affects the unit test, config.xml and system.xml):
```suggestion
    private const XML_PATH_ENABLED = 'media_gallery/general/enabled';
```